docs/en/week06/06-3.md

```diff
@@ -221,38 +211,77 @@ How do we turn off the output? Let’s say we have a purple internal representat
+<b>Figure 23:</b>Input Vector Example
+</center>
 
 
 2). Defining the Model and Training
 
-Let’s create a simple recurrent network, a LSTM and train for 10 epoches. In the training loop, we should always look for five steps: 1) perform the forward pass of the model, 2) compute the loss, 3) zero the gradient cache, 4) backpropagate to compute the partial derivative of loss with regard to parameters, 5) step in the opposite direction of gradient.
+Let’s create a simple recurrent network, a LSTM and train for 10 epoches. In the training loop, we should always look for five steps:
+* Perform the forward pass of the model
+* Compute the loss
+* Zero the gradient cache
+* Backpropagate to compute the partial derivative of loss with regard to parameters
+* Step in the opposite direction of gradient
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/train_test_easy.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 24:</b>Simple RNN vs LSTM - 10 Epochs
+</center>
 
 With easy level of difficulty, RNN gets 50% accuracy while LSTM gets 100% after 10 epoches. But LSTM has four times more weights than RNN and has two hidden layers, so it is not a fair comparison. After 100 epoches, RNN also gets 100% accuracy, but it takes longer than LSTM.
 
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/train_test_hard.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 25:</b>Simple RNN vs LSTM - 100 Epochs
+</center>
+
 If we increase the difficulty of the training part (using longer sequences), we will see the RNN fails while LSTM keeps working.
 
-The above visualization is drawing the value of hidden state over time in LSTM. We will send the inputs through a hyperbolic tangent, such that if the input is below -2.5, it will be mapped to -1, and if it is above 2.5, it will be mapped to 1. So in this case, we can see the specific hidden layer picked on X (fifth row in the picture) and then it became red until we got the other X. So, the fifth hidden unit of the cell is triggered by observing the X and goes quiet after seeing the other X. This allows us to recognize the class of sequence.
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/hidden_state_lstm.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 26:</b>Visualization of Hidden State Value
+</center>
+
+The above visualization is drawing the value of hidden state over time in LSTM. We will send the inputs through a hyperbolic tangent, such that if the input is below -2.5, it will be mapped to -1, and if it is above 2.5, it will be mapped to 1. So in this case, we can see the specific hidden layer picked on $X$ (fifth row in the picture) and then it became red until we got the other $X$. So, the fifth hidden unit of the cell is triggered by observing the $X$ and goes quiet after seeing the other $X$. This allows us to recognize the class of sequence.
```

Comment:
```suggestion
The above visualization is drawing the value of hidden state over time in LSTM. We will send the inputs through a hyperbolic tangent, such that if the input is below $-2.5$, it will be mapped to $-1$, and if it is above $2.5$, it will be mapped to $1$. So in this case, we can see the specific hidden layer picked on `X` (fifth row in the picture) and then it became red until we got the other `X`. So, the fifth hidden unit of the cell is triggered by observing the `X` and goes quiet after seeing the other `X`. This allows us to recognize the class of sequence.
```